ner/src/main/java/edu/illinois/cs/cogcomp/ner/IO/ResourceUtilities.java

```diff
@@ -37,14 +37,7 @@ public static InputStream loadResource(String resourceFile) {
             // jar)
             if (!new File(file).exists()) {
                 logger.debug("Loading {} from classpath", resourceFile);
-                List<URL> list = IOUtils.lsResources(ResourceUtilities.class, resourceFile);
```

Comment:
if there is a problem with the IOUtils.lsResources() method, please create an issue and give an example